flyteadmin/run.sh

```diff
@@ -0,0 +1,2 @@
+dlv --listen=:2345 --headless=true --api-version=2 --accept-multiclient exec bin/flyteadmin serve -- --config  flyteadmin_config.yaml 
```

Comment:
should we remove this file?

Reply:
I  use this for debugging using delve remote debugger. Removed it though and keeping it only for my local